client/src/components/ColorList.js

```diff
@@ -17,14 +17,27 @@ const ColorList = ({ colors, updateColors }) => {
   };
 
   const saveEdit = e => {
-    e.preventDefault();
+    // e.preventDefault();
     // Make a put request to save your updated color
     // think about where will you get the id from...
     // where is is saved right now?
+    axiosWithAuth()
+      .put(`http://localhost:5000/api/colors/${colorToEdit.id}`, colorToEdit)
```

Comment:
Because we're editing a specific color called `colorToEdit`, we want our `PUT` request to use the id from our specific `colorToEdit`.